libraries/botbuilder-dialogs/src/prompts/choicePrompt.ts

```diff
@@ -20,18 +23,10 @@ import { Culture } from '@microsoft/recognizers-text';
 export class ChoicePrompt extends Prompt<FoundChoice> {
 
     /**
-     * Default options for rendering the choices to the user based on locale.
+     * A dictionary of Default Choices based on [[PromptCultureModels.getSupportedCultures()]].
+     * Can be replaced by user using the constructor that contains choiceDefaults.
      */
-    private static defaultChoiceOptions: { [locale: string]: ChoiceFactoryOptions } = {
```

Comment:
This is technically a breaking change. We should discuss...

Reply:
resolved per email. property is private